src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -318,7 +331,9 @@ void GeneralPage::initLayout(GeneralPageView &layout)
     layout.addSeperator();
     layout.addCheckbox("Draw a line below the most recent message before "
                        "switching applications.",
-                       s.showLastMessageIndicator);
+                       s.showLastMessageIndicator, false,
+                       "Adds a red underline below the most recent message "
```

Comment:
Line color is configurable, having this mention red even when it's not certain might be confusing.